teku/src/main/java/tech/pegasys/teku/cli/options/P2POptions.java

```diff
@@ -150,11 +154,25 @@ public String getP2pPrivateKeyFile() {
   }
 
   public int getP2pLowerBound() {
-    return p2pLowerBound;
+    if (p2pLowerBound > p2pUpperBound) {
+      LOG.warn(
+          "Adjusting target number of peers lower bound to equal upper bound, which is {}",
+          p2pUpperBound);
```

Comment:
This will get lost in the log file but probably should go to the console instead (via `StatusLogger`).

Reply:
Done.